atd-vze/src/Components/GridExportData.js

```diff
@@ -169,6 +144,19 @@ const GridExportData = ({
       return item;
     });
 
+    const cleanedFlattenedAndParsedData = flattenedAndParsedData.map(item => {
```

Comment:
moved this to take place after we parse out the columns we dont need to export so we arent cleaning columns that arent even going into the csv

Reply:
Great call!